transformations/use_acronyms/test.json

```diff
@@ -0,0 +1,61 @@
+{
+  "type": "use_acronyms",
+  "test_cases": [
+
+    {
+      "class": "UseAcronyms",
+      "inputs": {
+        "sentence": "I studied at New York University and Massachusetts Institute of Technology."
+      },
+      "outputs": [
+        {
+          "sentence": "I studied at NYU and MIT."
+        }
+      ]
+    },
+    {
+      "class": "UseAcronyms",
+      "inputs": {
+        "sentence": "They changed my connection flight from Newark Liberty International Airport to Los Angeles International Airport."
+      },
+      "outputs": [
+        {
+          "sentence": "They changed my connection flight from EWR to LAX."
+        }
+      ]
+    },
+    {
+      "class": "UseAcronyms",
+      "inputs": {
+        "sentence": "My brother works at the Center for Science in the Public Interest."
+      },
+      "outputs": [
+        {
+          "sentence": "My brother works at the CSPI."
+        }
+      ]
+    },
+    {
+      "class": "UseAcronyms",
+      "inputs": {
+        "sentence": "Electro-Optical Tactical Sensor are a fundamental part of the system."
+      },
+      "outputs": [
+        {
+          "sentence": "EOTS are a fundamental part of the system."
+        }
+      ]
+    },
+    {
+      "class": "UseAcronyms",
+      "inputs": {
+        "sentence": "A strong ElectroMagnetic Pulse will damage nearby electronic devices."
+      },
+      "outputs": [
+        {
+          "sentence": "A strong EMP will damage nearby electronic devices."
```

Comment:
Why not do the reverse also since you have access to this large dictionary? Or set a default parameter in the constructor to do so selectively.

Reply:
Hi, the problem is that an acronym might represent different sets of words (eg. IMRI can be "Information Management Research Institute" ,"International Market Research Information", or "Intraoperative Magnetic Resonance Imaging"). Because of that the transformation Acronym -> Set of Words is not trivial. Not even sure it's solvable as of right now.

Reply:
Aah! I guessed that would be the case! Cool, I will merge this! Thanks for the clarification!